domains.txt

```diff
@@ -617,27 +685,34 @@ sjit.co.uk
 skala.co.uk
 skillrecruit.com
 skillsearch.com
+skypath.co.uk
```

Comment:
JOB SECTORS
We are travel technology recruitment specialists, recruiting the tech, digital and marketing specialists changing how we travel.